test/e2e/config/azure-dev.yaml

```diff
@@ -83,29 +83,29 @@ providers:
   - name: azure
     type: InfrastructureProvider
     versions:
-    - name: v1.12.2 # latest patch of earliest minor in supported v1beta1 releases; this is used for v1beta1 old --> v1beta1 latest clusterctl upgrades test only.
-      value: https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases/download/v1.12.2/infrastructure-components.yaml
+    - name: v1.13.2 # latest patch of earliest minor in supported v1beta1 releases; this is used for v1beta1 old --> v1beta1 latest clusterctl upgrades test only.
+      value: https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases/download/v1.13.2/infrastructure-components.yaml
       type: url
       contract: v1beta1
       files:
       - sourcePath: "../data/shared/v1beta1_provider/metadata.yaml"
-      - sourcePath: "../data/infrastructure-azure/v1.12.2/cluster-template-prow.yaml"
+      - sourcePath: "../data/infrastructure-azure/v1.13.2/cluster-template-prow.yaml"
         targetName: "cluster-template.yaml"
       replacements:
       - old: "imagePullPolicy: Always"
         new: "imagePullPolicy: IfNotPresent"
-    - name: v1.13.0 # latest patch of latest minor in supported v1beta1 releases; this is used for v1beta1 latest --> v1beta1 current clusterctl upgrades test only.
-      value: https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases/download/v1.13.0/infrastructure-components.yaml
+    - name: v1.14.0 # latest patch of latest minor in supported v1beta1 releases; this is used for v1beta1 latest --> v1beta1 current clusterctl upgrades test only.
+      value: https://github.com/kubernetes-sigs/cluster-api-provider-azure/releases/download/v1.14.0/infrastructure-components.yaml
       type: url
       contract: v1beta1
       files:
       - sourcePath: "../data/shared/v1beta1_provider/metadata.yaml"
-      - sourcePath: "../data/infrastructure-azure/v1.13.0/cluster-template-prow.yaml"
+      - sourcePath: "../data/infrastructure-azure/v1.13.2/cluster-template-prow.yaml"
```

Comment:
Should this be `"../data/infrastructure-azure/v1.14.0/cluster-template-prow.yaml"` ?